config/update_software_lists.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# This script is used to do the following:
+# - Normalize each of the data/cisagov_*.yml files.
+# - Update the comprehensive data/cisagov.yml file.
+# - Generate each software_lists/software_list_*.md file from its respective
+#   data/cisagov_*.yml file.
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+OUTPUT_DIRECTORY=software_lists
+TEMPLATE_FILE=config/SOFTWARE-LIST.tpl.md
+
+echo Normalize individual cisagov YAML files
+for file in data/cisagov_*.yml; do
+  echo "  $file..."
+  normalize-yml --cisagov-format "$file" > "$file".tmp
+  mv --force "$file".tmp "$file"
+done
+
+echo Update the comprehensive cisagov YAML file
+normalize-yml --cisagov-format data/cisagov_*.yml > data/cisagov.yml
+
+echo Generate Markdown files from the individual cisagov YAML files
+for file in data/cisagov_*.yml; do
+  echo "  $file..."
+  # Convert the file path data/cisagov_*.yml to software_list_*.md
+  md_file=$(echo "$file" | sed 's/data\/cisagov_\(.\+\)yml/software_list_\1md/g')
+  normalize-yml "$file" > "$file.tmp"
```

Comment:
Is it necessary to re-run `normalize-yml` here?

Reply:
It is because we need the output of `normalize-yml` without the `--cisagov-format` option for `yml2md` to correctly process the YAML file. If `normalize-yml` were rewritten to natively interpret a fully formatted YAML file this would no longer be necessary.

Reply:
OK, that makes sense.